### ISSUE TITLE
Rendering of the login link in a reduced window

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1083]
## Bug Report

**Problematic behavior**
With a screen 1005 x 584px wide, I navigate using the keyboard to access the login menu placed on the right side of the screen. The screen reader returns an untranslated term "Autenticate" (which is in the login url) and which takes me to the Visio login page.

**Expected behavior/code**
When I log in, I must be able to access the login menu in the language chosen for my interface and it must be rendered in the correct language. The link does not have an explicit label.

The code in question:
`<a target="_blank" class="flex rounded rounded-1 justify-center items-center text-base w-full md:w-fit text-left gap-[7px] px-4 h-10 transition ease-in-out delay-50 duration-300 font-medium text-brand-550 bg-transparent hover:bg-[#f2f5f4] active:bg-[#f2f5f4] md:!py-2 md:!px-4 h-8 !p-2 !min-w-0" href="[https://visio.numerique.gouv.fr/api/v1.0/authenticate?silent=false&amp;returnTo=https%3A%2F%2Fvisio.numerique.gouv.fr%2F](https://visio.numerique.gouv.fr/api/v1.0/authenticate?silent=false&amp;returnTo=https%3A%2F%2Fvisio.numerique.gouv.fr%2F)"><span aria-hidden="true"><svg xmlns="[http://www.w3.org/2000/svg](http://www.w3.org/2000/svg)" width="18" height="18" viewBox="0 0 18 18" fill="none" aria-hidden="true"><path d="M9 18V16H16V2H9V0H16C16.55 0 17.0208 0.195833 17.4125 0.5875C17.8042 0.979167 18 1.45 18 2V16C18 16.55 17.8042 17.0208 17.4125 17.4125C17.0208 17.8042 16.55 18 16 18H9ZM7 14L5.625 12.55L8.175 10H0V8H8.175L5.625 5.45L7 4L12 9L7 14Z" fill="currentColor"></path></svg></span><span></span></a>`

**Steps to Reproduce**
1. Open a window with a viewport of 1005 x 584px
2. Log in to Visio
3. And then the bug happens!

**Additional context/Screenshots**

<img width="985" height="257" alt="Image" src="https://github.com/user-attachments/assets/2d05ad1a-64d8-4e57-b35a-438c9dee89a6" />

**Comment by lebaudantoine** [comment]
Could you please reopen this issue on the landing page website? 

This page is not implemented on this repo.
